src/stories/Library/multiselect/Multiselect.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { Multiselect as MultiselectComp } from "./Multiselect";
+
+export default {
+  title: "Library / Dropdown / Multiselect",
+  component: MultiselectComp,
+  decorators: [withDesign],
+  argTypes: {
+    options: {
+      defaultValue: ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5"],
```

Comment:
We should stick to one language. I think the convention is that all texts in the design system should be danish. Or am I wrong?

Reply:
I am pretty sure it's English.
![image](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/28546954/6229d7ca-681b-4211-aa0d-3f83223525bb)
![image](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/28546954/051f1265-a0b4-4f7a-ae2b-f430d8bca595)

Reply:
Ok let's stick to english if that is what is widely used! 👍